source/manual/add-a-new-document-type.html.md

```diff
@@ -6,60 +6,49 @@ layout: manual_layout
 section: Publishing
 ---
 
-The [document type] describes what a page on GOV.UK looks like.
+The [document type][] describes what a page on GOV.UK looks like.
 
-## Add the document type in the govuk-content-schema repo
+[document type]: https://docs.publishing.service.gov.uk/document-types.html
 
-You need to add the document type into the
-[allowed document types][allowed-document-types] in
-[govuk-content-schemas][govuk-content-schemas].
+## Add the document type in the govuk-content-schema repo
 
-Once you have added the document type you should:
+You need to add the document type into the [allowed document types][] in [govuk-content-schemas][]. Once you have added the document type you should:
 
-- commit the change
-- run `bundle exec rake` to generate the schemas again
-- commit this update separately
+- Commit the change
+- Run `bundle exec rake` to generate the schemas again
+- Commit this update separately
 
 Examples of implementation:
 
-- [https://github.com/alphagov/govuk-content-schemas/pull/652][pr-652]
-- [https://github.com/alphagov/govuk-content-schemas/pull/630][pr-630]
+- <https://github.com/alphagov/govuk-content-schemas/pull/652>
+- <https://github.com/alphagov/govuk-content-schemas/pull/630>
 
-## Create a new rake task in the publishing app
+[allowed document types]: https://github.com/alphagov/govuk-content-schemas/blob/main/lib/govuk_content_schemas/allowed_document_types.yml
+[govuk-content-schemas]: https://github.com/alphagov/govuk-content-schemas
 
-The rake task will publish the route using the new document type.
+## Add a new content schema
 
-Before you begin, you must identify which publishing and rendering applications
-will use the document type. You need to include these as parameters (payload)
-when you generate the route.
+If your document type needs a new content schema, see "[Adding a new schema][]".
 
-Examples of implementation:
+[Adding a new schema]: https://github.com/alphagov/govuk-content-schemas/blob/main/docs/adding-a-new-schema.md
```

Comment:
```suggestion
[Adding a new schema]: /apps/govuk-content-schemas/adding-a-new-schema.html
```

Note to self: we should probably remove this confusing `app_name`: https://github.com/alphagov/govuk-developer-docs/blob/7c00e2cb356cd2e9af496baf6f48c26d740e9f67/data/applications.yml#L411